SampleTest/MainPage.xaml

```diff
@@ -21,10 +19,15 @@
         After finishing the initial registration page, you will also need to add an additional redirect URI.
         Click on "Add a Redirect URI" and check the https://login.microsoftonline.com/common/oauth2/nativeclient checkbox on that page. Then click "Save".
     -->
-    <Interactivity:Interaction.Behaviors>
+    <!--<Interactivity:Interaction.Behaviors>-->
```

Comment:
If this is working, you can remove this stuff here and remove the NuGet reference to the Behaviors package too.

Reply:
Will do :)